src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -188,6 +191,17 @@ public boolean validate() {
             return false;
         }
 
+        //null element in the values of all the possible keys with DataType as ARRAY
+        for (Entry<String, DataType> allowedKey : allowedKeys.entrySet()) {
+            DataType dataType = allowedKey.getValue();
+            JsonNode value = contentAsNode.get(allowedKey.getKey());
+            if (dataType == DataType.ARRAY && value != null) {
```

Comment:
Remove check for `dataType`. It may be a json node that has json array as an element. With nested check for `null` elements in json array it is not necessary to rely on `dataType`.

Reply:
Done

Reply:
Done